releasenotes/notes/add-instruction-supported-6323e618f04ae77e.yaml

```
@@ -0,0 +1,41 @@
+---
+features_transpiler:
+  - |
+    Add functions ``qk_target_instruction_supported``, ``qk_target_reset_supported``, and
+    ``qk_target_measure_supported`` to check the compatibility of a ``QkGate``, a :class:`.Reset`
+    instruction, or a :class:`.Measure` with the instance of ``QkTarget``
```

```suggestion
    instruction, or a :class:`.Measure` with the instance of :cpp:class:`QkTarget`.
```
This will probably not make `QkTarget` a real cross reference but at least it'll make look it like one.

---

I will use `:c:type:` instead, see https://www.sphinx-doc.org/en/master/usage/domains/c.html.